docs/security/trust/trust_key_mng.md

```diff
@@ -0,0 +1,63 @@
+<!--[metadata]>
++++
+title = "Manage keys for trust"
+description = "Manage keys for trust"
+keywords = ["trust, security, root,  keys, repository"]
+[menu.main]
+parent= "smn_content_trust"
++++
+<![end-metadata]-->
+
+# Manage keys for trust
+
+Trust for an image tag is managed through the use of keys. Docker's content
+trust makes use four different keys: 
+
+| Key                 | Description                                                                                                                                                                                                                                                                                                                                                                         |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| offline key            | Root of trust for a image tag. When trust is enabled, you create this key once. Then, you can use this key to create tagging keys.  |
+| target and snapshot | These two keys are known together as the repository pair.  When trust is enabled, you create this key when you add a new image repository.  If you have the offline key, you can export the tagging key and allow other images creators to sign image tags.       |
+| timestamp           | This key applies to a repository. It allows Docker repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side.  Each tag is signed with a tagging key and a timestamp key.                                                                                                                |
+
+With the exception of timestamps, all the keys are generated and stored locally
+client-side. The timestamp is generated and stored in a Docker registry. 
+
+## Choosing a passphrase
+
+The passphrase you chose for both the offline key and your tagging key should be
```

Comment:
Maybe add: `This passphrase is be used to encrypt your keys at rest, ensuring that a lost laptop or an unintended backup doesn't put the private key material at risk`